edenia/antelope-tools
# Feat/improve loading speed 1108

### Fix bugs in main page and backend

### What does this PR do?

- Resolve #1108
- Fix Telos producer's graph
- Fix eosrate permissions
- Fix hapi variable redeclaration in getEOSIORewards

### Steps to test

1. Run the project locally with telos mainnet
1. Check the main page
